From 9671431fa07f80c4b64385ba671cd120f4ba5930 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 8 Sep 2014 15:07:39 -0400
Subject: [ethernet] ixgbevf: add tx counters

Message-id: <1410188883-6595-25-git-send-email-jogreene@redhat.com>
Patchwork-id: 92364
O-Subject: [RHEL7.1 PATCH 24/48] ixgbevf: add tx counters
Bugzilla: 1091124
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: 1091124
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

This patch adds counters for tx_restart_queue and tx_timeout_count.

Signed-off-by: Alexander Duyck <alexander.h.duyck@intel.com>
Signed-off-by: Emil Tantilov <emil.s.tantilov@intel.com>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 6e2580f9a4d92611ea46583c270ac22403e87515)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbevf/ethtool.c b/drivers/net/ethernet/intel/ixgbevf/ethtool.c
index bff931f..9534e04 100644
--- a/drivers/net/ethernet/intel/ixgbevf/ethtool.c
+++ b/drivers/net/ethernet/intel/ixgbevf/ethtool.c
@@ -77,6 +77,8 @@ static const struct ixgbe_stats ixgbe_gstrings_stats[] = {
  {"tx_bytes", IXGBEVF_STAT(stats.vfgotc, stats.base_vfgotc,
       stats.saved_reset_vfgotc)},
  {"tx_busy", IXGBEVF_ZSTAT(tx_busy)},
+ {"tx_restart_queue", IXGBEVF_ZSTAT(restart_queue)},
+ {"tx_timeout_count", IXGBEVF_ZSTAT(tx_timeout_count)},
  {"multicast", IXGBEVF_STAT(stats.vfmprc, stats.base_vfmprc,
        stats.saved_reset_vfmprc)},
  {"rx_csum_offload_errors", IXGBEVF_ZSTAT(hw_csum_rx_error)},
-- 
1.7.1